source/server/admin/profile.proto

```diff
@@ -0,0 +1,212 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Would it be possible to use profile.proto from the google/pprof repo instead?

Reply:
I can try to bring it in via custom workspace import, `google/pprof` itself has no bazel rules on its own

Reply:
Use of an external BUILD file like some of the ones defined at https://github.com/envoyproxy/envoy/tree/main/bazel/external could help here at least until a BUILD file is added to the google/pprof repo.